kernel_gateway/gatewayapp.py

```diff
@@ -144,9 +146,23 @@ def _default_kernel_name_default(self):
         # defaults to Jupyter's default kernel name on empty string
         return os.getenv(self.default_kernel_name_env, '')
 
+    list_kernels_env = 'KG_LIST_KERNELS'
+    list_kernels = Bool(config=True,
+        help='Controls whether /api/kernels will report running kernels and their IDs (KG_LIST_KERNELS env var)'
+    )
+    def _list_kernels_default(self):
+        return os.getenv(self.list_kernels_env, 'False') == 'True'
+
+    api_env = 'KG_API'
+    api = Unicode(config=True,
+        help='Designates that the kernel gateway will serve APIs from this notebook (KG_API env var)'
+    )
+    def _api_default(self):
+        return os.getenv(self.api_env)
```

Comment:
Probably want this returning a default string representing the "notebook kernel API + jupyter protocol over websocket" option. Why: we've got one protocol+api, you guys are working on a second (notebook defined API over HTTP), and there might be more coming down the pipe (e.g., see #17). Best to plan for extensibility here.

Make the help print the available valid options. You can get it to show the default too: see the others where I set the value + used the default method.